magicctapipe/resources/test_config_general.yaml

```diff
@@ -0,0 +1,28 @@
+mc_tel_ids:
+    LST-1: 1
+    LST-2: 0
+    LST-3: 0
+    LST-4: 0
+    MAGIC-I: 2
+    MAGIC-II: 3
+
+directories:
```

Comment:
all those directories below as well as the LST and MAGIC runs are only relevant for the second (Torino_IT_container) PR, right? If so, they should be also kept only there and removed from here

Reply:
Yes, we removed all lines that are relevant only for the next PR from the configuration files test_config_general.yaml and test_config_general_4LST.yaml